en/ui-system/components/editor/canvas.md

```diff
@@ -0,0 +1,31 @@
+# Canvas Component Reference
+
+The **Canvas** component can obtain the actual resolution of the device screen at any time and zoom in / out of all the rendered elements in the scene. There can be multiple Canvas in the scene, and all UI elements must be placed under Canvas and its child nodes to be rendered.
+The Canvas design resolution and adaptation scheme are uniformly set in the **Project Setting**. A camera is provided inside the Canvas, and the default z axis direction is **-1000 ~ 998**, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
+
+![](canvas/canvas.png)
+
+In the previous design, Canvas was last rendered, meaning it could mask the rendering of all 3D content, but this was far from sufficient for project development needs, such as a 2D map with a 3D character. So, we've added the **RenderMode** property, which allows developers to sort the rendering of the 3D Camera and UI Camera by changing the behavior of the UI Camera, which needs to be used with **ClearFlag**.
```

Comment:
... So, we've added the **RenderMode** property, which allows developers to sort the rendering order of the 3D Camera and UI Camera. To be noted, if you want to have canvas and 3d camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.